src/modules/vtol_att_control/tiltrotor.cpp

```diff
@@ -266,35 +267,91 @@ void Tiltrotor::update_external_state()
 
 }
 
- /**
-* Prepare message to acutators with data from the attitude controllers.
+/**
+* Write data to actuator output topic.
 */
-void Tiltrotor::fill_att_control_output()
+void Tiltrotor::fill_actuator_outputs()
 {
-	_actuators_out_0->control[0] = _actuators_mc_in->control[0] * _roll_weight_mc;	// roll
-	_actuators_out_0->control[1] = _actuators_mc_in->control[1] * _roll_weight_mc;	// pitch
-	_actuators_out_0->control[2] = _actuators_mc_in->control[2] * _roll_weight_mc;	// yaw
+	switch(_vtol_schedule.flight_mode) {
```

Comment:
I fully agree on the suggestion about the enum, I can do that. Also I don't see any reason why we can't rename the actuator struct.